{{cookiecutter.repo_name}}/devtools/conda-envs/test_env.yaml

```diff
@@ -1,21 +1,28 @@
-name: test
-channels:
-{% if cookiecutter.dependency_source == 'Prefer conda-forge over the default anaconda channel with pip fallback' %}
-  - conda-forge
-{% endif %}
+name: {{cookiecutter.repo_name}}-test
+channels: {% if cookiecutter.dependency_source == 'Prefer conda-forge over the default anaconda channel with pip fallback' %}
+  - conda-forge {% endif %}
   - defaults
 dependencies:
     # Base depends
   - python
   - pip
 
+    # MDAnalysis
+  - cython
```

Comment:
you shouldn't need this downstream, MDA should pull from setup.py or pyproject.toml, if it doesn't that's an us problem